_config.yml

```diff
@@ -30,13 +30,6 @@ owner:
   email:
   twitter: bitcoincoreorg
   facebook: #username
-  google:
-    plus: #username
-    analytics: #UA-85008990-1
-    verify:
-    ad-client:
-    ad-slot:
-  bing-verify:
   github: bitcoin
   stackoverflow: #123456/username   from a "http://stackoverflow.com/users/123456/username" link
   linkedin: #username
```

Comment:
I guess if we're deleting unnecessary settings, there's quite a few below here as well which could go (all of them? I don't see a bitcoin core steam release happening any time soon…).